crates/server/src/store/file/authority.rs

```diff
@@ -30,6 +30,7 @@ use crate::store::in_memory::InMemoryAuthority;
 ///
 /// Authorities default to DNSClass IN. The ZoneType specifies if this should be treated as the
 /// start of authority for the zone, is a Secondary, or a cached zone.
+#[cfg_attr(not(any(feature = "openssl", feature = "dnssec-ring")), derive(Clone))]
```

Comment:
just make this `#[derive(Clone)]` it's not clear why it should only be when there is no openssl/dnssec-ring support.

Reply:
Cause Signer is not clone if ring or OpenSSL is enabled.

Reply:
This is probably an oversight. I think the OpenSSL types are cloneable, but the *ring* types are not in Signer. We can put the *ring* types behind an Arc to make that cloneable, which is what is recommended here:  https://github.com/briansmith/ring/issues/859#issuecomment-605129968